behaviours/__init__.py

```diff
@@ -7,14 +7,15 @@
 from behaviours import notalurker, jimmy_nerfer, messageparty
 from behaviours import userinfo
 from behaviours import meme_repost_blocker, meme_rater, rant_patrol, paidnotpayed
-from behaviours import snark, deletes, duplicate_message_policing
+from behaviours import snark, deletes, duplicate_message_policing, fight_club
 
 _Evt = TypeVar("_Evt", bound=hikari.Event)
 _Chain = Sequence[Sequence[Callable[[_Evt], Coroutine[None, None, None]]]]
 
 _message_create_chain: _Chain[hikari.GuildMessageCreateEvent] = [
     # Message filtering & deletion
     [
+        fight_club.main,
```

Comment:
This should go lower, probably in the generic responses section immediately before or after paidnotpayed. And it should raise EndProcessing when appropriate.